components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/handler/AdminForcedPasswordResetHandler.java

```diff
@@ -207,6 +207,25 @@ private void lockAccountOnAdminPasswordReset(User user, UserStoreManager userSto
         setUserClaims(userClaims, user, userStoreManager);
     }
 
+    private void publishPreAdminPasswordResetEvent( Map<String, Object> map) throws IdentityEventException {
```

Comment:
`TriggerPreAdminForcedPasswordReset` WDYT?

Reply:
Fixed in 61d7548fc71aca19ecb53cb10928da2bb00294d6